packages/web/src/components/HomeButtons.tsx

```diff
@@ -104,57 +116,64 @@ export const routerLinkButtons: THomeButtonRouterLink[] = [
 
 const coordinatedEntryButton: THomeButtonRouterLink = {
   text: "Coordinated Entry",
+  translationKey: "home:coordinatedEntry",
   icon: DoorIcon,
   linkProps: {
     to: "/coordinated-entry",
   },
   color: colors.rosewood,
 };
 
-const HomeButtons = () => (
-  <Grid
-    container
-    direction="row"
-    justify="space-evenly"
-    alignItems="stretch"
-    spacing={2}
-  >
-    {routerLinkButtons.map(button => {
-      return (
-        <Grid item xs={6} key={button.text}>
-          <Box
-            alignContent="stretch"
-            component={HomeRouterLink}
-            display="flex"
-            height="100%"
-            {...button}
-          >
-            <HomeButton
-              buttonColor={button.color}
-              data-test="home-router-button"
+const HomeButtons = () => {
+  const { t } = useTranslation(["home"]);
+  return (
+    <Grid
+      container
+      direction="row"
+      justify="space-evenly"
+      alignItems="stretch"
+      spacing={2}
+    >
+      {routerLinkButtons.map(button => {
+        return (
+          <Grid item xs={6} key={button.text}>
+            <Box
+              alignContent="stretch"
+              component={HomeRouterLink}
+              display="flex"
+              height="100%"
+              {...button}
             >
-              {button.text}
-              {button.icon}
-            </HomeButton>
-          </Box>
-        </Grid>
-      );
-    })}
-    <Grid item xs={12}>
-      <Box
-        alignContent="stretch"
-        component={HomeRouterLink}
-        display="flex"
-        height="100%"
-        {...coordinatedEntryButton}
-      >
-        <HomeButton buttonColor={coordinatedEntryButton.color}>
-          {coordinatedEntryButton.text}
-          {coordinatedEntryButton.icon}
-        </HomeButton>
-      </Box>
+              <HomeButton
+                buttonColor={button.color}
+                data-test="home-router-button"
+              >
+                {t(button.translationKey, button.text)}
```

Comment:
So is `button.text` here acting as a backup if a translation key has not been supplied? I'm having trouble finding documentation on since the [second argument looks like it's supposed to be options](https://www.i18next.com/overview/api#t).

Reply:
Yeah, I meant to use it as a backup. Here's the [documentation reference](https://www.i18next.com/translation-function/essentials#passing-a-default-value). It looks like this also works, and is more intention revealing `t(button.translationKey, { defaultValue: button.text })`.

Reply:
Ah sweet! Ya either was good with me but just couldn't find where that was 😅